lib/parse.js

```diff
@@ -7,6 +7,6 @@ function parse (profile, graph, uri, mimeType, callback) {
     $rdf.parse(profile, graph, uri, mimeType)
     return callback(null, graph)
   } catch(e) {
-    return callback('Cound not load/parse profile data')
+    return callback('Cound not load/parse profile data', e)
```

Comment:
Shouldn't a callback return an instance of Error, in this case?
So, that line should be:

```js
return callback(new Error('Could not load/parse profile data'))
```

Also, do the unit tests pass for you, with this change? For some reason they're failing on Travis.